decidim-results/db/migrate/20170612101951_migrate_results_category.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateResultsCategory < ActiveRecord::Migration[5.1]
+  def change
+    Decidim::Results::Result.find_each do |result|
```

Comment:
Oh right! I was trying to find a way to share the code implied in all that. But I guess that's kind of unnecessary.